includes/class-relme-widget.php

```diff
@@ -77,4 +79,21 @@ public function form( $instance ) {
 		esc_html_e( 'Displays rel=me links which appear as icons with the logo of the site linked to when possible', 'indieweb' );
 		echo '</p>';
 	}
+
+	/**
+	 * Fetch the first administrator ID.
+	 *
+	 * @return mixed Administrator user ID.
+	 */
+	public function get_default_admin_author() {
+		$users = get_users(
+			array(
+				'role'   => 'administrator',
+				'number' => 1,
+				'fields' => 'ids',
+			)
+		);
+
+		return $users[0];
```

Comment:
The doku says, that it returns a `stdClass`, shouldn't the function return `$users[0]->ID` in this case?

Reply:
If we're fetching just IDs, it doesn't return objects of just IDs, it'll just be an array of ID values.